api/lib/domain/models/CertificationCourse.js

```diff
@@ -3,6 +3,12 @@ const Joi = require('joi').extend(require('@joi/date'));
 const { EntityValidationError } = require('../errors');
 
 const ABORT_REASONS = ['candidate', 'technical'];
+const cpfImportStatus = {
+  ERROR: 'ERROR',
+  GENERATED: 'GENERATED',
+  PENDING: 'PENDING',
+  SUCCESS: 'SUCCESS',
```

Comment:
Je me demande si c'est tres clair, possibilité:
```suggestion
  CPF_ERROR: 'CPF_ERROR',
  READY_TO_SEND: 'READY_TO_SEND',
  PENDING: 'PENDING',
  CPF_SUCCESS: 'CPF_SUCCESS',
```

Reply:
On est dans la colonne ```cpfImportStatus``` ça paraissait redondant de remettre ```cpf``` par contre j'aime bien le READY_TO_SEND